squirrel.api/pom.xml

```diff
@@ -99,31 +105,33 @@
             <groupId>org.aksw.jena-sparql-api</groupId>
             <artifactId>jena-sparql-api-core</artifactId>
         </dependency>
-        
-		        <!-- https://mvnrepository.com/artifact/org.rdfhdt/hdt-java-package -->
-		
-		<dependency>
-		    <groupId>org.rdfhdt</groupId>
-		    <artifactId>hdt-java-package</artifactId>
-		    <type>pom</type>
-		    <exclusions>
-		        <exclusion>
-		          <groupId>junit</groupId> <!-- Exclude Project-D from Project-B -->
-		          <artifactId>junit-dep</artifactId>
-		        </exclusion>
-		      </exclusions>
-		</dependency>
-        
-				 
+
+        <!-- https://mvnrepository.com/artifact/org.rdfhdt/hdt-java-package -->
+
+        <dependency>
+            <groupId>org.rdfhdt</groupId>
+            <artifactId>hdt-java-package</artifactId>
+            <type>pom</type>
+            <exclusions>
+                <exclusion>
+                    <groupId>junit</groupId> <!-- Exclude Project-D from Project-B -->
+                    <artifactId>junit-dep</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+
 
         <!-- HTTP Requests -->
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpcore</artifactId>
+            <version>4.4.11</version>
         </dependency>
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
+            <version>4.5.7</version>
```

Comment:
Why do we have the versions here, now? Shouldn't they be part of the parent pom?